app/models/comment.rb

```diff
@@ -1,4 +1,8 @@
 class Comment < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
+  def update_comments_counter
+    post.increment!(:CommentsCounter)
+  end
```

Comment:
- [ ] I'd suggest you use *one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) -* `after_save` to call the `#update_comments_counter` method every time a new post has been saved to database, then make the `update_comments_counter` method private. 
For example
```ruby
after_save :update_comments_counter

  private

def update_comments_counter
  post.increment!(:comments_counter)
end
```

Reply:
Thanks, I didn't know that! :+1: